src/TaskManager/Runner/Program.cs

```diff
@@ -60,7 +60,7 @@ private static async Task Main(string[] args)
             var wmConfig = host.Services.GetRequiredService<IOptions<WorkflowManagerOptions>>();
             Guard.Against.NullService(wmConfig, nameof(IOptions<WorkflowManagerOptions>));
 
-            subscriber.Subscribe(messagingKeys.TaskUpdateRequest, string.Empty, (args) =>
+            subscriber.Subscribe(messagingKeys.TaskUpdateRequest, "TaskUpdate", (args) =>

```

Comment:
I think this should be also messagingKeys.TaskUpdateRequest to keep it consistent